ci-operator/step-registry/baremetal/lab/post/add-nodes/baremetal-lab-post-add-nodes-commands.sh

```diff
@@ -0,0 +1,143 @@
+#!/bin/bash
+
+set -o nounset
+set -o pipefail
+
+CLUSTER_NAME="${NAMESPACE}"
+DIR=/tmp/bmo
+mkdir -p ${DIR}
+
+master00_addr=$(yq e '.[] | select(.name == "master-00") | .bmc_address' ${SHARED_DIR}/hosts.yaml)
+master00_bootMacAddr=$(yq e '.[] | select(.name == "master-00") | .provisioning_mac' ${SHARED_DIR}/hosts.yaml)
+master00_username=$(yq e '.[] | select(.name == "master-00") | .bmc_user' ${SHARED_DIR}/hosts.yaml)
+master00_pass=$(yq e '.[] | select(.name == "master-00") | .bmc_pass' ${SHARED_DIR}/hosts.yaml)
+
+workera_addr=$(yq e '.[] | select(.name == "worker-a-00") | .bmc_address' ${SHARED_DIR}/hosts.yaml)
+workera_bootMacAddr=$(yq e '.[] | select(.name == "worker-a-00") | .provisioning_mac' ${SHARED_DIR}/hosts.yaml)
+workera_username=$(yq e '.[] | select(.name == "worker-a-00") | .bmc_user' ${SHARED_DIR}/hosts.yaml)
+workera_pass=$(yq e '.[] | select(.name == "worker-a-00") | .bmc_pass' ${SHARED_DIR}/hosts.yaml)
+
+cat > ${DIR}/bmo.yaml <<EOF
+apiVersion: metal3.io/v1alpha1
+kind: Provisioning
+metadata:
+  name: provisioning-configuration
+spec:
+  provisioningNetwork: "Disabled"
+  watchAllNamespaces: false
+
+EOF
+
+cat > ${DIR}/master00-bmc.yaml <<EOF
```

Comment:
do we need to do this only for one master, or all?

Reply:
Only once